collectors/python.d.plugin/python.d.plugin.in

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-'''':; exec "$(command -v python || command -v python3 || command -v python2 ||
+'''':; exec "$(command -v python || command -v python3 || command -v python3.6 || command -v python2 || command -v python2.7 ||
```

Comment:
What will happen when user with FreeBSD decides to upgrade to newer python version? Should we add all python versions here just because FreeBSD cannot package python properly? Because if so, then it looks like a dead end.

Also `lang/python` in FreeBSD should create a symlink from `python2.7` to `python2`. Source: https://svnweb.freebsd.org/ports/head/lang/python/Makefile?view=markup&pathrev=491284#l30